Card Focus/CardFocus.js

```diff
@@ -70,25 +75,36 @@ tau.mashups
             },
 
             focusOnCards: function() {
-                var clipboardManager = configurator.getClipboardManager();
+                var clipboardManager = appConfigurator.getClipboardManager();
 
                 var cards = _.values(clipboardManager._cache);
 
-                var ids = cards.reduce(function(r, item) {
-                    r.push(item.data.id); return r;
-                }, []);
+                var ids = _.chain(cards)
+                    .filter(function(c) {
+                        return c.isSelected;
+                    })
+                    .map(function(c) {
+                        return c.data.id;
+                    })
+                    .value();
+
+                if (ids.length === 0) {
+                    return;
+                }
+
+                var filter = '?Id is ' + ids.join(' or Id is ');
```

Comment:
Too tricky, I guess, probably it would be better to map ids to `Id is <value>` and then join with or.